e2e/tests/omnibus-arbitrum-mainnet-arbthetix.toml/Liquidity_Provider.e2e.js

```diff
@@ -116,7 +116,7 @@ describe(require('path').basename(__filename, '.e2e.js'), function () {
     });
   });
```

Comment:
test for withdraw?

Reply:
Cannot withdraw if cannot deposit =)
Does not matter really. Because this is sunset of arbthetix and we will remove it from git after deployment.